src/lib/apiTypesV2/Entity.cpp

```diff
@@ -103,9 +103,23 @@ std::string Entity::check(ConnectionInfo* ciP, RequestType requestType)
     return "No Entity ID";
   }
 
-  if (forbiddenChars(id.c_str()))         { return "Invalid characters in entity id";        }
-  if (forbiddenChars(type.c_str()))       { return "Invalid characters in entity type";      }
-  if (forbiddenChars(isPattern.c_str()))  { return "Invalid characters in entity isPattern"; }
+  if (forbiddenChars(id.c_str()))
```

Comment:
Fixed in 8afc5d4

Finally I removed the LM_W (which was actually an LM_E!!!) from the lower level (function forbiddenChars()) and I made sure **all** callers invoke  LM_W(("Bad Input ()")).
Advantage: more info on a higher level.
Inconvenient: **every** time forbiddenChars is invoked, the LM_W(("Bad Input ()")) must be invoked.

Doing it on a higher level is more 'risky' and more work, but the end-result is better.